src/code/z_play.c

```diff
@@ -15,33 +15,42 @@ u64 D_801614D0[0xA00];
 
 void Play_SpawnScene(PlayState* this, s32 sceneNum, s32 spawn);
 
-void func_800BC450(PlayState* this) {
-    Camera_ChangeBgCamIndex(GET_ACTIVE_CAM(this), this->unk_1242B - 1);
+void Play_ChangeViewpointBgCamIndex(PlayState* this) {
+    Camera_ChangeBgCamIndex(GET_ACTIVE_CAM(this), this->viewpoint - 1);
 }
 
-void func_800BC490(PlayState* this, s16 point) {
-    ASSERT(point == 1 || point == 2, "point == 1 || point == 2", "../z_play.c", 2160);
+void Play_SetViewpoint(PlayState* this, s16 viewpoint) {
+    ASSERT(viewpoint == VIEWPOINT_LOCKED || viewpoint == VIEWPOINT_PIVOT, "point == 1 || point == 2", "../z_play.c",
+           2160);
 
-    this->unk_1242B = point;
+    this->viewpoint = viewpoint;
 
-    if ((YREG(15) != 0x10) && (gSaveContext.cutsceneIndex < 0xFFF0)) {
-        Audio_PlaySoundGeneral((point == 1) ? NA_SE_SY_CAMERA_ZOOM_DOWN : NA_SE_SY_CAMERA_ZOOM_UP, &gSfxDefaultPos, 4,
-                               &gSfxDefaultFreqAndVolScale, &gSfxDefaultFreqAndVolScale, &gSfxDefaultReverb);
+    if ((R_SCENE_CAM_TYPE != SCENE_CAM_TYPE_FIXED_SHOP_VIEWPOINT) && (gSaveContext.cutsceneIndex < 0xFFF0)) {
+        // C-Up toggle the camera inside a house
```

Comment:
```suggestion
        // Play a sound when toggling the camera inside a house with C-Up
```
not sure if mentioning C-Up is even worth here, it's just that it needs to not play a sound during the intro cutscene (z_demo, `CS_MISC(0x000E`)

Reply:
Yeah true, can take out the `C-Up`